### PR TITLE
Fixed Issue #8877

### PR DESCRIPTION
**Pull request by amit-webkul**

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@amit-webkul it is working fine.

**Comment by ashishkumar-webkul** [comment]
@amit-webkul As I can see that there are some conflicts appearing, Please resolve the conflicts.

File: packages/Webkul/Admin/src/Resources/views/emails/orders/canceled.blade.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
         </span> <br>
 
         <p style="font-size: 16px;color: #5E5E5E;line-height: 24px;">
-            @lang('admin::app.emails.dear', ['customer_name' => $order->customer_full_name]),👋
+            @lang('admin::app.emails.dear', ['admin_name' => $order->customer_full_name]),👋
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/emails/orders/canceled.blade.php:8]: Why we are passing customer name in "admin_name" parameter

File: packages/Webkul/Admin/src/Resources/views/emails/orders/invoiced.blade.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
         </span> <br>
 
         <p style="font-size: 16px;color: #5E5E5E;line-height: 24px;">
-            {{ __('admin::app.emails.dear', ['customer_name' => $invoice->order->customer_full_name]) }},👋
+            {{ __('admin::app.emails.dear', ['admin_name' => $invoice->order->customer_full_name]) }},👋
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/emails/orders/invoiced.blade.php:8]: Why we are passing customer name in "admin_name" parameter

File: packages/Webkul/Admin/src/Resources/views/emails/orders/refunded.blade.php [hunk 3]
```diff
@@ -5,7 +5,7 @@
         </span> <br>
 
         <p style="font-size: 16px;color: #5E5E5E;line-height: 24px;">
-            {{ __('admin::app.emails.dear', ['customer_name' => $refund->order->customer_full_name]) }},👋
+            {{ __('admin::app.emails.dear', ['admin_name' => $refund->order->customer_full_name]) }},👋
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/emails/orders/refunded.blade.php:8]: Why we are passing customer name in "admin_name" parameter

**Comment by jitendra-webkul** [comment]
Conflicts

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**